TCC.Lib/Process/ProcessStartHelper.cs

```diff
@@ -0,0 +1,169 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Management;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace TCC.Lib.Process
+{
+    public static class ProcessStartHelper
+    {
+        public static Task<CommandResult> Run(this string command, string workingDirectory,
+            CancellationToken cancellationToken, int timeoutMinutes = 480)
+        {
+            return command.Run(workingDirectory, cancellationToken, TccProcessOutputClassifier.Instance, timeoutMinutes);
+        }
+
+        public static Task<CommandResult> Run(this string command, string workingDirectory,
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 22 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS3776)